src/main/java/com/example/javaOjt/services/AuthorService.java

```diff
@@ -59,10 +65,32 @@ public GetAuthorResponse getAuthorById(Integer id, boolean withBooks) {
   }
 
   /**
-   * Fetches the complete list of authors.
+
+   * Fetches the list of authors and sort if needed.
```

Comment:
```suggestion
   * Fetches the list of authors and sort if needed.
   *
   * @param attribute the attribute to sort by (id, name)
   * @param order the order to sort by (asc, dsc)
   * @return GetAuthorsResponse containing the list of authors
```

javadoc の `@param` と `@return` で parameter と戻り値の説明を入れるのは一般的なので書きましょう！